schemas/rum/_common-schema.json

```diff
@@ -300,6 +300,27 @@
             }
           }
         },
+        "configuration": {
+          "type": "object",
+          "description": "Subset of the configuration options used to initialize the SDK",
+          "readOnly": true,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** 

- description: FMU, we should send a value even if the customer use the default option
- required fields: if configuration is present, we should always have session sample rate, not sure for session replay rate on mobile, it could worth double checking with them

‏
```suggestion
        "configuration": {
          "type": "object",
          "description": "Subset of the configuration options used by the SDK",
          "required": ["session_sample_rate"]
          "readOnly": true,
```

Reply:
I reworded the description, let me know what you think.
I made both fields as required since there is a session replay sample rate for mobile too.